versions/3.0.4.md

```diff
@@ -1765,9 +1750,13 @@ Note also that `Content-Transfer-Encoding` is deprecated for `multipart/form-dat
 Using `format: byte` for a multipart field is equivalent to setting `Content-Transfer-Encoding: base64`.
 If `format: byte` is used along with setting a different `Content-Transfer-Encoding` value with the `headers` field, the result is undefined.
```

Comment:
Can we say this instead?
```suggestion
If `format: byte` is specified for a `multipart/form-data` field, the result is undefined.
```

Reply:
No, because it is completely valid to do this.  "Deprecated" does not mean "forbidden", and the behavior is completely well-defined.  In RFC7578 it is a SHOULD NOT not a MUST NOT, and that RFC was only published in July 2017 – the same month as OAS 3.0.  We support all HTTP APIs, including ones that use behavior that is now deprecated.

The text I wrote in this paragraph provides all of the necessary context for authors to decide whether it is suitable for their use or not.  We wouldn't want to _replace_ this sentence with a `form-data`-specific sentence anyway as it's at least theoretically possible to describe other `multipart` types.